internal/restore/volumesnapshot_action.go

```diff
@@ -85,6 +86,9 @@ func (p *VolumeSnapshotRestoreItemAction) Execute(input *velero.RestoreItemActio
 			Annotations: map[string]string{
 				velerov1api.RestoreNameLabel: input.Restore.Name,
 			},
+			Labels: map[string]string{
```

Comment:
do we need it as an annotation as well? (lines just above) If we're trying to match what Velero does, it only adds a label. Note that it also adds the backup name label, we might want to do that too.

Reply:
Removing the annotation should be OK.

I don't think we want to add the backup name label because we use that to identify resources to delete when backups expire.